### DISCUSSION TITLE
I cant use nginx to listen to port 3000

### DISCUSSION THREAD

**Discussion by SepehrMoafi** [nuxt/nuxt#16422] [Questions]
I want to run a web site with Nuxt js 3 ( 3.0.0-rc.12 ) on ubuntu ( 20.04 ) vps , I can see the web site with ``IP:3000`` but I can not see on ``mydomain.com``  , I installed nginx and run this commands to Start npm :
1. ```npm run generate```
2. ```npm run build```
3. ```node .output/server/index.mjs```

this commands will start npm on ```Listening http://[::]:3000``` 

This is my nginx config 

```
server {
    listen         80;
    server_name      mydomain.com  www.mydomain.com 

    gzip            on;
    gzip_types      text/plain application/xml text/css application/javascript;
    gzip_min_length 1000;

    location / {
        proxy_pass http://localhost:3000;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection 'upgrade';
        proxy_set_header Host $host;
        proxy_cache_bypass $http_upgrade;
    }
}
```
Sorry for ask this question im new in this field

**Comment by sourcecodeit** [comment]
Have the same issue. Running the Nuxt application from the public IP:PORT doesn't have issues but if I load it through Nginx on DOMAIN:80 I have this issue:

`entry.d442b30e.js:1 Hydration completed but contains mismatches.`

I can't understand what's wrong :(
